dbusmock/templates/systemd-base.py

```diff
@@ -0,0 +1,91 @@
+'''systemd mock template base
+'''
+
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 3 of the License, or (at your option) any
+# later version.  See http://www.gnu.org/copyleft/lgpl.html for the full text
+# of the license.
+
+__author__ = 'Jonas Ådahl'
+__copyright__ = '(c) 2021 Red Hat'
+
+from gi.repository import GLib
+
+BUS_NAME = 'org.freedesktop.systemd1'
+MAIN_OBJ = '/org/freedesktop/systemd1'
```

Comment:
Maybe you could use these as basic prefix for avoiding repetitions below?

Reply:
Added prefixes for the name and path, and changed to use that in the rest of the file.

Reply:
Using prefixes now.